src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -42,47 +44,55 @@ export type Props<L> = {
   /**
    * An optional class to apply to the dropdown.
    */
-  dropdownClassName?: string;
+  dropdownClassName?: string | null;
+  /**
+   * Additional props to pass to the dropdown.
+   */
+  dropdownProps?: SubComponentProps<ContextualMenuDropdownProps>;
```

Comment:
@Caleb-Ellis what do you think about this method of passing props to child components?

Reply:
Seems alright to me